driver-service/src/main/java/com/dulfinne/taxi/driverservice/mapper/DriverMapper.java

```diff
@@ -1,15 +1,19 @@
 package com.dulfinne.taxi.driverservice.mapper;
 
+import com.dulfinne.taxi.driverservice.dto.request.DriverRequest;
 import com.dulfinne.taxi.driverservice.dto.response.DriverResponse;
 import com.dulfinne.taxi.driverservice.model.Driver;
 import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
 import org.mapstruct.factory.Mappers;
 
 @Mapper
```

Comment:
Тогда можно пользоваться возможностями спринга и внедрять маппер в качестве зависимости там, где он требуется (например, в сервис). Статики для таких вещей лучше избегать.
```suggestion
@Mapper(componentModel = MappingConstants.ComponentModel.SPRING)
```